packages/web/src/App.tsx

```diff
@@ -1,8 +1,7 @@
 import React from "react";
 import { Layout } from "antd";
 import "./App.css";
-import { Route, Switch } from "react-router-dom";
-
+import { Switch } from "react-router-dom";
```

Comment:
I assume we want to import this switch also from "react-router-dom-v5-compat" and completely remove the package "react-router-dom"

Reply:
Yup will work on this as well. Thank you!